dbfit-java/core/src/main/java/dbfit/fixture/InspectProcedure.java

```diff
@@ -0,0 +1,9 @@
+package dbfit.fixture;
+
+public class InspectProcedure extends Inspect
+{

```

Comment:
That `{` should be at the end of the previous line (as per our current Java style guidelines https://github.com/dbfit/dbfit/blob/master/CONTRIBUTING.md). Similar remark applies to some of the other changes.